spec/requests/users_onboarding_spec.rb

```diff
@@ -13,17 +13,17 @@
       end
 
       it "updates the attributes on the user" do
```

Comment:
```suggestion
      it "updates the user's last_onboarding_page attribute" do
```
Since this test doesn't actually update any _other_ user attributes anymore (like location) because those are all extracted out of the user and onto the profile fields, we may want to consider changing this.